components/collective-page/sections/RecurringContributions.js

```diff
@@ -37,7 +37,7 @@ const I18nFilters = defineMessages({
     defaultMessage: 'Yearly',
   },
   [FILTERS.CANCELLED]: {
-    id: 'Subscriptions.Cancelled',
+    id: 'Subscriptions.Canceled',
```

Comment:
Not sure why we're updating the `id` here? It will force translators to re-translate this for all languages. Great if we can keep it like that.

Reply:
🤦🏼